tests/zenko_e2e/backbeat/Using.md

```diff
@@ -10,34 +10,44 @@
    ```
 
 2. Create an account using Orbit.
-3. Export the access key and secret key of that account:
+3. Export the access key and secret key of that account (for example, in
+   `.secrets.env`):
 
 ```
-export ZENKO_BACKBEAT_ACCESS_KEY=<access-key>
-export ZENKO_BACKBEAT_SECRET_KEY=<secret-key>
+export ZENKO_STORAGE_ACCOUNT_ACCESS_KEY=<access-key>
+export ZENKO_STORAGE_ACCOUNT_SECRET_KEY=<secret-key>
 ```
 
-3. Install node and npm.
-4. Navigate to `Zenko/tests/zenko_e2e/backbeat`.
-5. Install node modules: `npm i`.
+4. Install node and npm.
+5. Navigate to `Zenko/tests/zenko_e2e/backbeat`.
+6. Install node modules: `npm i`.
+7. If using `.secrets.env`, source the file: `source .secrets.env`.
 
 ### Tests for CRR to AWS:
 
 1. Create a bucket on AWS `<destination-bucket-name>` with versioning enabled.
 2. In Orbit, create an AWS location `<destination-location-name>` with an AWS
    `<destination-bucket-name>`.
 3. In Orbit, create an AWS location `<source-location-name>`.
-4. Export the access key, secret key, AWS bucket name, and AWS location:
+4. Export the access key, secret key, bucket and container names, and storage
+   location names (for example, in `.env` and `.secrets.env`):
 
 ```
-export AWS_S3_BACKBEAT_ACCESS_KEY=<access-key>
-export AWS_S3_BACKBEAT_SECRET_KEY=<secret-key>
-export AWS_S3_BACKBEAT_BUCKET_NAME=<destination-bucket-name>
-export AWS_DESTINATION_LOCATION=<destination-location-name>
-export AWS_SOURCE_LOCATION=<source-location-name>
+export AWS_S3_BACKEND_ACCESS_KEY=<access-key>
+export AWS_S3_BACKEND_SECRET_KEY=<secret-key>
+export AWS_S3_BACKBEAT_BUCKET_NAME=<destination-aws-bucket-name>
+export AWS_S3_BACKEND_DESTINATION_LOCATION=<destination-aws-location>
+export AWS_S3_BACKEND_SOURCE_LOCATION=<source-aws-location>
+export AZURE_BACKEND_ACCOUNT_NAME=<azure-account-name>
+export AZURE_BACKEND_ACCESS_KEY=<access-key>
+export AZURE_BACKEND_ENDPOINT=<endpoint>
+export AZURE_BACKBEAT_CONTAINER_NAME=<destination-azure-container-name>
+export AZURE_BACKEND_DESTINATION_LOCATION=<destination-azure-location>
 ```
 
 5. Run the test suite: `npm run test_crr`.
+7. If using `*.env` files, source the files:
+   `source .secrets.env && source .secrets.env`.
```

Comment:
Why do we need to source the secrets twice?

Reply:
You don't need to, of course. Since the guide we update the file in two different steps in the guide, I figured it would help ensure that it's not forgotten.

Reply:
I meant more so why `source .secrets.env && source .secrets.env` vs `source .secrets.env`